WalletWasabi.Documentation/FAQ.md

```diff
@@ -0,0 +1,181 @@
+FAQ for [Wasabi Wallet](https://github.com/zkSNACKs/WalletWasabi)
+By [6102bitcoin](https://twitter.com/6102bitcoin)
+
+Note: Many of these Q&A have been copied from real users, see footer for acknowledgements. 
+
+# Index
+- Pre-Install
+- Install
+- Pre-Mix
+- Mixing
+- Post-Mix
+- Meta
+- Acknowledgements
+
+# Pre-Install
+
+### Who is behind Wasabi?
+
+The Company that is developing Wasabi is zkSNACKs LTD ([twitter](@Zksnacks_LTD) | [website](https://zksnacks.com/))
+
+The following people are listed on the company website:
+
+| Name						| Twitter 						 																		| Position							|
+| ----------------| --------------------------------------------------------|	----------------------|
+| Hajdú Gergely 	|	[@gergely_hajdu](https://twitter.com/gergely_hajdu) 		| CEO & Co-Founder			|
+| Harmat Bálint		|	[@bharmat84](https://twitter.com/bharmat84) 						| CEO & Co-Founder			|
+| Adam Ficsor			|	[@nopara73](https://twitter.com/nopara73)								| CTO										|
+| Lucas Ontivero	| [@lontivero](https://twitter.com/)											| Lead Engineer					|
+
+Other people have contributed to the code:
+- David Molnar ([@molnardavid84](https://twitter.com/molnardavid84))
+- Dan Walmsley ([@danwalmsley](https://github.com/danwalmsley))
```

Comment:
Well, the listing there is just outdated, but oh well, not sure I want to nitpick this, but I will anyway:)

Instead of making recommendation here, I'll just resolve this section myself.